js/main-menu.js

```diff
@@ -122,7 +122,7 @@
    */
   function wrapMenuItemLabelInHTML ($menuItem) {
     $menuItem.contents().filter(function () {
-      return this.nodeType === 3 && jQuery(this.parentElement).is($menuItem);
+      return this.nodeType === 3 && jQuery(this.parentNode).is($menuItem);
```

Comment:
I think we should use, `jQuery(this).parent()`, which is cross browser and its jQuery, so no need to worry about browser compatibility.

Reply:
totally make sense, thanks